src/app/service/user.service.ts

```diff
@@ -0,0 +1,38 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { error } from 'protractor';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class UserService {
+  private link =" ";
+
+  constructor(private http: HttpClient) { }
+  
+  login(email: string , password: string) {
+    var postData= { email: email , password : password}
+    var post = this.http 
```

Comment:
Use `let` instead of `var`.
```suggestion
    let post = this.http 
```